lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -300,6 +310,37 @@ class QcFlagService {
         });
     }
 
+    /**
+     * Create verification of QC flag
+     * @param {Partial<QcFlagVerification>} parameters flag verification parameters
+     * @param {object} relations QC Flag entity relations
+     * @param {UserWithRoles} relations.userWithRoles user with roles
+     * @return {Promise<QcFlag>} promise
+     * @throws {NotFoundError|AccessDeniedError}
+     */
+    async verifyFlag(parameters, relations) {
```

Comment:
You can't name a parameter parameter... name it verification or whatever you want, but not parameters...

Reply:
renamed